client/scss/nav.scss

```diff
@@ -0,0 +1,36 @@
+@import 'https://fonts.googleapis.com/css?family=Press+Start+2P';
```

Comment:
here is an example of commenting on some code.
I clicked on "Files changed" and then clicked on the line I wanted to comment on. 

You can discuss things you're not sure about in this way